### PR TITLE
Patch 2019-10-31 new weapon and chara

### PR DESCRIPTION
**Pull request by kei-gbf**
This patch contains 2019-10-28 Event update.

## New Chara
- [x] [Societte (Wind)](https://gbf.wiki/Societte_(Wind))
- [x] [Kou](https://gbf.wiki/Kou)
  - [ ] XXX: check original data source, min ATK is not wrong paste?
- [x] EVENT [You (SR)](https://gbf.wiki/You_(Event))

## New Weapon
- [x] [Cardinal Spear](https://gbf.wiki/Cardinal_Spear)
- [x] [Juzumaru](https://gbf.wiki/Juzumaru)
- [x] EVENT [Holy Fox Strings](https://gbf.wiki/Holy_Fox_Strings)

Test Data: [2019-10-31.json.txt](https://github.com/MotocalDevelopers/motocal/files/3843890/2019-10-31.json.txt)

### CODE REVIEW

File: charaData.json [hunk 1]
```diff
@@ -5411,6 +5411,50 @@
...
+        "race": "erune",
+        "sex": "male",
+        "fav1": "katana",
+        "fav2": "music",
+        "support": "none",
+        "support2": "none",
+        "support3": "none",
+        "minhp": "240",
+        "hp": "1389",
+        "minattack": "1389",
```
Comment (kagekiyo7) [inline charaData.json]: > XXX: check original data source, min ATK is not wrong paste?

No.

image from internet
image

File: charaData.json [hunk 2]
```diff
@@ -11262,5 +11306,27 @@
...
         "imageURL": "./charaimgs/3030275000_01.png",
         "en": "La Coiffe"
+    },
+    "ヨウ": {
+        "name": "ヨウ",
+        "ja": "ヨウ",
+        "element": "dark",
+        "type": "attack",
+        "race": "erune",
+        "sex": "male",
```
Comment (kagekiyo7) [inline charaData.json]: >  "sex": "male",

female (ヨウ)

Comment (Groovman) [reply charaData.json]: That is wrong in the wiki :)

File: charaData.json (hunk 1 above)
Comment (kei-gbf) [reply charaData.json]: thx, ok I just doubt it coz that was same with maxHP

File: charaData.json (hunk 2 above)
Comment (kei-gbf) [reply charaData.json]: merged already. did anyone fix this or another PR?

Comment (kei-gbf) [reply charaData.json]: ok I found the answer on #384

Comment (kei-gbf) [reply charaData.json]: ok I found the answer on

**Pull request merged**